### PR TITLE
Clean up README.md a little

### PR DESCRIPTION
**Pull request by notgull**
Signed-off-by: John Nunley <dev@notgull.net>

### CODE REVIEW

**Review by madsmtm — APPROVED** [review]
What about https://github.com/rust-windowing/softbuffer/pull/79?

**Comment by notgull** [comment]
> What about #79?

This one has less diff noise.

File: README.md [hunk 1]
```diff
@@ -1,25 +1,29 @@
 Overview
 ==
-As the popularity of the library [minifb](https://crates.io/crates/minifb) shows, it is useful to put a 2D buffer/image
-on a window in a platform-independent way. Minifb's approach to doing window management itself, however, is problematic
+
+Enables software rendering via drawing an image straight to a window.
+
+Softbuffer integrates with the [raw-window-handle](https://crates.io/crates/raw-window-handle) crate
```
Comment (MarijnS95) [inline README.md:6]:
```suggestion
Softbuffer integrates with the [`raw-window-handle`](https://crates.io/crates/raw-window-handle) crate
```

File: README.md [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
+to allow writing to a window in a cross-platform way while using the very high quality dedicated window management
```
Comment (MarijnS95) [inline README.md:7]: I find "writing to a window" quite vague. Can we clarify this by adding "pixels" or "colors" or something?

**Comment by MarijnS95** [comment]
#79 has been left ignored by the submitter for 1.5 years, and seems to have quite some conflicts after the readme was improved more piecemeal on the `master` branch. We should just close it for lack of intent and actionability.

**Pull request merged**